app/views/cookies/show.en.html.erb

```diff
@@ -9,14 +9,20 @@
 
     <h1 class="govuk-heading-xl">Cookies</h1>
 
-    <p class="govuk-body"><%= service_name -%> puts small files (known as ‘cookies’) onto your device to collect
+    <p class="govuk-body">The '<%= service_name -%>' service puts small files (known as ‘cookies’) onto your device to collect
       information about how you use the service.</p>
 
-    <p class="govuk-body">We only set cookies when javascript is running in your browser and you’ve accepted them. If
-      you choose not to run javascript, the information on this page will not apply to you.</p>
+    <p class="govuk-body">You’ll normally see a message on the site before we set a cookie on your device.</p>
 
-    <p class="govuk-body">You’ll normally see a message on the site before we store a cookie on your device.</p>
-    <p class="govuk-body">Cookies aren’t used to identify you personally.</p>
+    <p class="govuk-body">We only set cookies when:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>you’ve accepted them</li>
+      <li>JavaScript is running in your browser</li>
+    </ul>
+
+    <p class="govuk-body"> If you choose not to run JavaScript, the information on this page will not apply to you.</p>
```

Comment:
Perhaps `If you choose not to run JavaScript, some of the information on this page may not apply to you.`

Reply:
`If you choose not to run JavaScript, we will only set the essential cookies as outlined below.`